### DISCUSSION TITLE
VfsFile.createZipFromTree() - How to add directory being zipped to the root of the zip?

### DISCUSSION THREAD

**Discussion by slickorange** [korlibs/korge#2282] [Q&A] [answered]
How can I zip a directory with createZipFromTree so that the directory being zipped is at the root of the zip?

Eg at the moment when I zip a target directory it zips it as `/Users/user/files/my_directory`. I don't want the whole tree in the zip, it should only have the files inside my_directory in the zip

**Comment by soywiz** [comment, marked as answer]
Does this work for you?

```kotlin
localVfs("/Users/user/files/my_directory").jail().createZipFromTree()
```

**Reply by slickorange** [reply]
Perfect! Thank you so much for the quick help!
I want to add that I realize Korge's focus is on being a game engine.. but the IO library is unlike anything available for Kotlin Multiplatform.. It really could stand as an amazing library on its own - it fills a huge gap in terms of KMP development.
